### PR TITLE
highlights Arabic comma?

### PR DESCRIPTION
**Pull request by Felienne**
Hi @thjazi39!

I think this is a right fix for #3103, but: 

1) can you check it 
2) can we make it more robust so that we have a list of comma's somewhere that can easily be changed in one place (more languages will have more comma's) 
3) can we make this file be uft8, it is really hard to read Arabic characters like this. I understand this is easier for you, but it is not easier for everyone else that reads a bit of Arabic.

### CODE REVIEW

File: highlighting/highlighting.json [hunk 1]
```diff
@@ -429,7 +429,7 @@
                     ]
                 },
                 {
-                    "regex": ",",
+                    "regex": ",|\u060c",
```
Comment (Felienne) [inline highlighting/highlighting.json:432]: this is really tricky to read!

**Comment by Felienne** [comment]
ok re point 3, I changed the encoding but I guess we will manually have to make the \u060c back into the arabic comma and while we are added also change the Tatweel. Can you do that?

**Comment by thjazi39** [comment]
> 2. can we make it more robust so that we have a list of comma's somewhere that can easily be changed in one place (more languages will have more comma's)

I think the best solution will be to add a key in the yaml files.

> 3. can we make this file be uft8, it is really hard to read Arabic characters like this. I understand this is easier for you, but it is not easier for everyone else that reads a bit of Arabic.

The file is already in UTF8, so it seems to me that if we put directly the symbol instead of the utf8 code it will still work (python works with it after)

**Comment by thjazi39** [comment]
> ok re point 3, I changed the encoding but I guess we will manually have to make the \u060c back into the arabic comma and while we are added also change the Tatweel. Can you do that?

ok

**Comment by Felienne** [comment]
> > 2. can we make it more robust so that we have a list of comma's somewhere that can easily be changed in one place (more languages will have more comma's)
> 
> I think the best solution will be to add a key in the yaml files.
> 
> > 3. can we make this file be uft8, it is really hard to read Arabic characters like this. I understand this is easier for you, but it is not easier for everyone else that reads a bit of Arabic.
> 
> The file is already in UTF8, so it seems to me that if we put directly the symbol instead of the utf8 code it will still work (python works with it after)

Yes I added an extra commit to add the encoding, because earlier file converted the symbol.

**Comment by Felienne** [comment]
> I think the best solution will be to add a key in the yaml files.

There already is a comma symbol in the yaml files of the keywords! 

https://github.com/Felienne/hedy/blob/6c42e7e46476a338f1d488e9e41282ec411b0088/content/keywords/ar.yaml#L21

You could gather those, and be sure to always allow for the En comma in all languages as we do for keywords.

**Comment by Felienne** [comment]
> > I think the best solution will be to add a key in the yaml files.
> 
> There already is a comma symbol in the yaml files of the keywords!
> 
> https://github.com/Felienne/hedy/blob/6c42e7e46476a338f1d488e9e41282ec411b0088/content/keywords/ar.yaml#L21
> 
> You could gather those, and be sure to always allow for the En comma in all languages as we do for keywords.

And clearly it would be ideal to do this for other symbols too but we can do that in a separate PR.

**Comment by Felienne** [comment]
I don't really understand this change, if it needs to go in, can you make a
PR?

**Comment by Felienne** [comment]
Closing this in favor of #3107

**Pull request closed without merging**